antiplag/constants.py

```diff
@@ -1,4 +1,6 @@
 CONTENT_TYPE_FILE = "multipart/form-data"
-CONTENT_TYPE_TEXT = "text/plain"
+CONTENT_TYPE_JSON = "application/json"
```

Comment:
As discussed on retrospective, please rename both constants to `FILE_UPLOAD_CONTENT_TYPE` and `TEXT_UPLOAD_CONTENT_TYPE`.